.github/workflows/tests.yml

```diff
@@ -24,15 +24,27 @@ jobs:
       with:
         repository: fireeye/capa-testfiles
         path: tests/data
-    - name: Set up Python 3.8
+    - name: Set up Python 2.7
       uses: actions/setup-python@v2
       with:
-        python-version: 3.8
-    # We don't need vivisect, so we can install capa using Python3
+        python-version: 2.7
     - name: Install capa
       run: pip install -e .
-    - name: Run rule linter
-      run: python scripts/lint.py rules/
+    - name: Get modified files
+      id: files
+      uses: jitterbit/get-changed-files@v1
+    - name: Get modified rule names
+      run: |
+        for changed_file in ${{ steps.files.outputs.added_modified }}; do
+          tags+="\"$(grep 'name:' "${changed_file}" | sed 's/^.*: //')\" "
+        done
+        echo "$tags"
+        echo "tags=$tags" >> $GITHUB_ENV
+    - name: Run rule linter thoroughly
+      run: python scripts/lint.py -t ${{ env.tags }} rules/
+    # TODO removeme?
+#    - name: Run rule linter
+#      run: python scripts/lint.py rules/
```

Comment:
lets keep these around in non-thorough mode - only takes a few seconds to run i think